packages/firestore/src/remote/serializer.ts

```diff
@@ -659,31 +655,34 @@ export function fromMutation(
     ? fromPrecondition(proto.currentDocument)
     : Precondition.none();
 
+  const fieldTransforms = proto.updateTransforms
+    ? proto.updateTransforms.map(transform =>
+        fromFieldTransform(serializer, transform)
+      )
+    : undefined;
```

Comment:
Doesn't this need be initialized to `[]`?

Reply:
Changed.